clicommand/step_cancel.go

```diff
@@ -63,10 +66,17 @@ var StepCancelCommand = cli.Command{
 		},
 		cli.BoolFlag{
 			Name:   "force",
-			Usage:  "Don't wait for the agent to finish before cancelling the jobs",
+			Usage:  "Transition unfinished jobs to a canceled state instead of waiting for jobs to finish uploading artifacts",
 			EnvVar: "BUILDKITE_STEP_CANCEL_FORCE",
 		},
 
+		cli.Int64Flag{
+			Name:   "force-grace-period",
+			Value:  defaultCancelGracePeriod,
+			Usage:  "The number of seconds to wait for agents to finish uploading artifacts before transitioning unfinished jobs to a canceled state. Should be used in conjunction with `--force`",
+			EnvVar: "BUILDKITE_FORCE_GRACE_PERIOD,BUILDKITE_CANCEL_GRACE_PERIOD",
```

Comment:
Do we think `BUILDKITE_FORCE_GRACE_PERIOD` is the right name?

In isolation, it doesn't really describe what it's used for - not ideal for a global ENV var.

Perhaps it should be something like `BUILDKITE_FORCE_CANCEL_GRACE_PERIOD`?

```suggestion
			EnvVar: "BUILDKITE_FORCE_CANCEL_GRACE_PERIOD,BUILDKITE_CANCEL_GRACE_PERIOD",
```